### DISCUSSION TITLE
Display pega-embed with sdk

### DISCUSSION THREAD

**Discussion by oussamaInnovationTeam** [pegasystems/react-sdk#481] [General]
we create widget 
we add  this code to the component
  return (
    <StyledSirarComponentLibSirarGoIFrameWidgetWrapper>
      <React.Fragment>
        <pega-embed
          id="theEmbed"
          action="openPage"
          pageID="SelfServices"
          pageClass="Sirar-SelfServ-UIPages"
          casePage="full"
          appAlias="selfservice"
          pegaServerUrl="https://zonedev.sirar.com.sa:8080/prweb/"
          staticContentUrl="https://release.constellation.pega.io/8.23.0-110/react/prod/"
          grantType="none"
        ></pega-embed>
      </React.Fragment>
    </StyledSirarComponentLibSirarGoIFrameWidgetWrapper>
  );
publish the widget
add the script to imdex.html
it is works but it is intiate the redux  store and change the context

**Comment by timjmartel** [comment]
At this time the technology supports one active version of the ConstellationJS orchestration.  From the description it's a not clear what your trying to achieve.  Embedding constellation UI in an alternate SDK UI?
